Palace.xcodeproj/project.pbxproj

```diff
@@ -4022,7 +4022,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0.15;
```

Comment:
Could you revert it back to 1.0.15 please? We discussed last Monday that we change build number only for Test Flight builds; we change project version for builds that go to App Store.